docs/source/tutorials/otx_cli/how_to_train/detection.rst

```diff
@@ -0,0 +1,389 @@
+
+  To modify the architecture of supported models with various backbones, please refer to the :doc:`advanced tutorial for model customization <../../advanced/backbones>`.
+
+.. code-block::
+
+  (detection) ...$ otx find --template --task DETECTION
+  +-----------+-----------------------------------+-------+---------------------------------------------------------------------------+
+  |    TASK   |                 ID                |  NAME |                                    PATH                                   |
+  +-----------+-----------------------------------+-------+---------------------------------------------------------------------------+
+  | DETECTION |   Custom_Object_Detection_YOLOX   | YOLOX | otx/algorithms/detection/configs/detection/cspdarknet_yolox/template.yaml |
+  | DETECTION |  Custom_Object_Detection_Gen3_SSD |  SSD  |  otx/algorithms/detection/configs/detection/mobilenetv2_ssd/template.yaml |
+  | DETECTION | Custom_Object_Detection_Gen3_ATSS |  ATSS | otx/algorithms/detection/configs/detection/mobilenetv2_atss/template.yaml |
+  +-----------+-----------------------------------+-------+---------------------------------------------------------------------------+
+
+2. ``otx train`` trains a model (a particular model template) on a dataset and results in two files:
+
+- ``weights.pth`` - a model snapshot
+- ``label_schema.json`` - a label schema used in training, created from a dataset
+
+These are needed as inputs for the further commands: ``export``, ``eval``,  ``optimize``,  ``deploy`` and ``demo``.
+
+
+3. To have a specific example in this tutorial, all commands will be run on the ATSS model. It's a medium model, that achieves relatively high accuracy, while keeping the inference fast.
+
+The following command line starts 1 GPU training of the medium object detection model on BCCD dataset:
+
+.. code-block::
+
+  (detection) ...$ otx train otx/algorithms/detection/configs/detection/mobilenetv2_atss/template.yaml
+                            --train-ann-files data/wgisd/coco_annotations/train_bbox_instances.json 
+                            --train-data-roots  data/wgisd/data
+                            --val-ann-files data/wgisd/coco_annotations/test_bbox_instances.json 
+                            --val-data-roots data/wgisd/data 
+                            --save-model-to outputs
+                            --work-dir outputs/logs
+                            --gpus 1
+
+If you created ``data.yaml`` file in previous step, you can simplify the training by passing it in ``--data`` parameter:
+
+.. code-block::
+
+  (detection) ...$ otx train otx/algorithms/detection/configs/detection/mobilenetv2_atss/template.yaml
+                            --data data.yaml
+                            --save-model-to outputs
+                            --work-dir outputs/logs
+                            --gpus 1
```

Comment:
This might mean GPU ID 1, not using 1 GPU. @eunwoosh Could you confirm?

Reply:
You're right, will fix that